packages/data-service/package.json

```diff
@@ -74,6 +74,7 @@
     "@types/whatwg-url": "^8.2.1",
     "bson": "^5.2.0",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
```

Comment:
We refactored this package away from a lot of tests long time ago because it would cause tests to hang without throwing sometimes, just FYI if you start seeing these tests flake or get stuck without resolving. In other places we usually use try catch and expect.fail for asserting rejection